cookbook/email/email.rst

```diff
@@ -123,7 +123,21 @@ an email is pretty straightforward::
     }
 
 To keep things decoupled, the email body has been stored in a template and
-rendered with the ``renderView()`` method.
+rendered with the ``renderView()`` method. The ``registration.html.twig``
+template might look something like this:
+
+.. code-block:: html+jinja
+
+    {# app/Resources/views/Emails/registration.html.twig #}
+    <h3>You did it! You registered!</h3>
+
+    {# example, assuming you have a route named "login" $}
+    To login, go to: {{ url('login') }}.
```

Comment:
A minor comment: in the rest of this template we use HTML tags. Maybe we should replace `{{ url('login') }}` by `<a href="{{ url('login') }}">...</a>` to create a real link?